examples/profile-secret.yaml

```diff
@@ -0,0 +1,28 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Atlas AWS CloudFormation for building a, secret profile with ApiKeys
+Parameters:
+  ProfileName:
+    Description: "Your MongoDB Cloud Public API Key"
+    Type: String
+    Default: "default"
+  PublicKey:
+    Description: "Your MongoDB Cloud Public API Key"
+    Type: String
+    Default: "PublicKey"
+  PrivateKey:
+    Description: "Your MongoDB Cloud Private API Key"
+    Type: String
+    Default: "PrivateKey"
+Resources:
+  AtlasApiKeySecret:
+    Type: 'AWS::SecretsManager::Secret'
+    Properties:
+      Name: !Sub "cfn/atlas/profile/${ProfileName}"
+      Description: MongoDB Atlas API Key
+      #
+      # { "PublicKey" : "xxx", "PrivateKey", "yyy", "OrgId": "zzz" }
+      #
```

Comment:
[q] Anyway you can include the example in the description test? If multiline/code isn't supported no point, just a nice to have.